<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_window</class>
 <widget class="QDialog" name="settings_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>128</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>190</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="main_layout">
     <item>
      <layout class="QGridLayout" name="separators_layout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="output_separator_text"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="input_separator_text"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Input separator</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Output separator</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lower_layout">
       <item>
        <widget class="QCheckBox" name="dark_theme_button">
         <property name="text">
          <string>Dark theme (requires restart)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="accept_cancel_layout">
         <item>
          <widget class="QPushButton" name="accept_button">
           <property name="text">
            <string>Accept</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancel_button">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
